en/docs/develop/apis/identity-governance-rest-api.md

```diff
@@ -15,112 +15,86 @@ and properties of connectors as mentioned below. <br>
     
     | Property display name | Property name | Purpose |
     | ------- | -------|---|
-    | Enable Password History Feature   | passwordHistory.enable   | Enable to disallow previously used passwords |
-    | Password History validation count   | passwordHistory.count   | Restrict reusing last x number of passwords 
-    when updating the password |
+    | Validate password history   | passwordHistory.enable   | User will not be allowed to use previously used passwords. |
```

Comment:
Remove fullstops for all phrases.

Reply:
these were added as this is how it is returned in the API response.